similarity/config/test.configurations.properties

```diff
@@ -0,0 +1,20 @@
+memorybasedESA	src/test/resources/MemoryBasedESA_test.txt
+pageIDMapping	src/test/resources/wikiPageIDMapping_test.txt
+word2vec	src/main/resources/word2vec.txt
```

Comment:
are these resource files -- like word2vec -- stubs, or does the user have to download the data sets first?